html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Chained.pm

```diff
@@ -33,6 +33,11 @@ has_field 'authentication_source' =>
    options_method => \&options_authentication_source,
   );
 
+has_field authentication_source_can_continue => (
```

Comment:
I'm really not sure what this does...
A help message would be really useful here